services/dkg/pedersen/handler.go

```diff
@@ -477,3 +507,16 @@ func (h *Handler) checkIsShuffled(K kyber.Point, C kyber.Point, electionId strin
 	return false, nil
 
 }
+
+// MarshalJSON returns a JSON-encoded bytestring containing all the data in the Handler
+// that is meant to be persistent. It allows for saving the data to disk.
+func (h *Handler) MarshalJSON() ([]byte, error) {
```

Comment:
Why not just making `HandlerData` part of the Handler ?

Reply:
the question remains unanswered

Reply:
Since `HandlerData` only exists for marshalling, it seems reasonable to leave it separate from `Handler`